package.json

```diff
@@ -8,9 +8,11 @@
     "gh-pages": "^1.1.0",
     "perseids-react-components": "^0.0.2",
     "prop-types": "^15.6.2",
-    "react": "^16.4.0",
+    "react": "16.8.4",
```

Comment:
Is there a reason you pinned the exact version of `react`, `react-dom`, and `react-router-dom`?

Reply:
I was having issues adding react-router-i18n, so I set these to the ones listed on the react-router-i18n page. It may not have been necessary, but I did not have it working otherwise.